src/gui/widgets/LcdFloatSpinBox.cpp

```diff
@@ -240,9 +243,9 @@ void LcdFloatSpinBox::paintEvent(QPaintEvent*)
 	{
 		p.setFont(pointSizeF(p.font(), 6.5));
 		p.setPen(m_wholeDisplay.textShadowColor());
-		p.drawText(width() / 2 - horizontalAdvance(p.fontMetrics(), m_label) / 2 + 1, height(), m_label);
+		p.drawText(width() / 2 - p.fontMetrics().boundingRect(m_label).width() / 2 + 1, height(), m_label);
```

Comment:
If we're getting rid of `horizontalAdvance` can you check if we can remove `#include "DeprecationHelper.h"` too?

Reply:
On second thought, `QWheelEvent::position()` says "introduced in Qt 5.14", so it's probably better to keep the deprecation wrapper in use for now. `boundingRect()` should be fine, it existed in Qt 4.8. at the very least.